File: src/test/java/info/mastera/leetcode/validAnagram242/SolutionTest.java
package info.mastera.leetcode.validAnagram242;

import org.junit.jupiter.api.Assertions;
import org.junit.jupiter.api.Test;

class SolutionTest {

    @Test
    public void test() {
        Assertions.assertTrue(new Solution().isAnagram("anagram", "nagaram"));
        Assertions.assertFalse(new Solution().isAnagram("rat", "car"));
        Assertions.assertFalse(new Solution().isAnagram("aat", "tta"));
    }
}

File: src/test/java/info/mastera/leetcode/validParentheses20/SolutionTest.kt
package info.mastera.leetcode.validParentheses20


import org.junit.jupiter.api.Assertions.assertFalse
import org.junit.jupiter.api.Assertions.assertTrue
import org.junit.jupiter.api.Test

class SolutionTest {

    @Test
    fun test() {
        assertTrue(Solution().isValid("()"))
        assertFalse(Solution().isValid(")("))
        assertTrue(Solution().isValid("(())"))
        assertTrue(Solution().isValid("()[]{}"))
        assertFalse(Solution().isValid("(]"))
        assertTrue(Solution().isValid("{[]}"))
        assertFalse(Solution().isValid("()))"))
        assertFalse(Solution().isValid("["))
    }
}